corehq/pillows/mappings/group_mapping.py

```diff
@@ -1,59 +1,60 @@
-GROUP_INDEX="hqgroups_2g4222ug7mdk69tkh76p48oju38hre0k"
-GROUP_MAPPING= {
+GROUP_INDEX = "hqgroups_htj2o87ep2eug2g3k46hh2a67tn0r8d9"
+GROUP_MAPPING = {
     "date_formats": [
-        "yyyy-MM-dd", 
-        "yyyy-MM-dd'T'HH:mm:ssZZ", 
-        "yyyy-MM-dd'T'HH:mm:ss.SSSSSS", 
-        "yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'", 
-        "yyyy-MM-dd'T'HH:mm:ss'Z'", 
-        "yyyy-MM-dd'T'HH:mm:ssZ", 
-        "yyyy-MM-dd'T'HH:mm:ssZZ'Z'", 
-        "yyyy-MM-dd'T'HH:mm:ss.SSSZZ", 
-        "yyyy-MM-dd'T'HH:mm:ss", 
-        "yyyy-MM-dd' 'HH:mm:ss", 
-        "yyyy-MM-dd' 'HH:mm:ss.SSSSSS", 
+        "yyyy-MM-dd",
+        "yyyy-MM-dd'T'HH:mm:ssZZ",
+        "yyyy-MM-dd'T'HH:mm:ss.SSSSSS",
+        "yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'",
+        "yyyy-MM-dd'T'HH:mm:ss'Z'",
+        "yyyy-MM-dd'T'HH:mm:ssZ",
+        "yyyy-MM-dd'T'HH:mm:ssZZ'Z'",
+        "yyyy-MM-dd'T'HH:mm:ss.SSSZZ",
+        "yyyy-MM-dd'T'HH:mm:ss",
+        "yyyy-MM-dd' 'HH:mm:ss",
+        "yyyy-MM-dd' 'HH:mm:ss.SSSSSS",
         "mm/dd/yy' 'HH:mm:ss"
-    ], 
-    "dynamic": False, 
+    ],
+    "dynamic": False,
     "_meta": {
-        "comment": "Ethan created on 2014-02-11", 
+        "comment": "Ethan created on 2014-02-11",
         "created": None
-    }, 
-    "date_detection": False, 
+    },
+    "date_detection": False,
     "properties": {
         "doc_type": {
-            "index": "not_analyzed", 
+            "index": "not_analyzed",
             "type": "string"
-        }, 
+        },
         "domain": {
             "fields": {
                 "domain": {
-                    "index": "analyzed", 
+                    "index": "analyzed",
                     "type": "string"
-                }, 
+                },
                 "exact": {
-                    "index": "not_analyzed", 
+                    "index": "not_analyzed",
                     "type": "string"
                 }
-            }, 
+            },
             "type": "multi_field"
-        }, 
+        },
         "name": {
             "fields": {
                 "exact": {
-                    "index": "not_analyzed", 
-                    "type": "string"
-                }, 
+                    "index": "analyzed",
```

Comment:
Hey @esoergel it's already possible to sort not_analyzed exact fields, how come this was made into a text field and then we added a sortable_exact? (don't worry if you don't remember it's a very old PR). I ran into an issue with this analyzer in ES7, which is why I am pining here!

Reply:
I have no recollection of this, but it looks like the main thing is the `"filter": ["lowercase"]` above.  This probably used to sort "apple Banana carrot" as "Banana apple carrot", and using this analyzer allows for case-insensitive sorting.